caluma/caluma_workflow/models.py

```diff
@@ -236,3 +249,16 @@ class Meta:
             GinIndex(fields=["assigned_users"]),
             GinIndex(fields=["meta"]),
         ]
+
+
+@receiver(post_init, sender=WorkItem)
+def set_name_and_description(sender, instance, **kwargs):
+    """
+    Ensure WorkItem has a name and description set.
+
+    Default to values from Task.
+    """
+    if not any((value for value in instance.name.values())):
```

Comment:
You don't need the `value for value in` bit. `any(instance.name.values())` works just the same :)

Reply:
:man_facepalming:  Done.